update-imported-docs/README-hi.md

```diff
@@ -0,0 +1,61 @@
+# अपडेटेड इम्पोर्टेड डॉक्स
+
+यह स्क्रिप्ट डॉक्स फ़ाइलों को अपडेट करती है जो अन्य रीपॉज़टॉरी से उत्पन्न होती हैं।
+यह अपने इनपुट के रूप में एक YAML फ़ाइल नाम को स्वीकार करता है जिसे प्रति-रेपो आधार पर अनुकूलित किया जा सकता है।
+
+## आवश्यकताएँ
+
+इम्पोर्टेड डॉक्स को इन अनुदेशों का पालन करना चाहिए:
+
+1. [प्रलेखन शैली गाइड](/docs/home/contribute/style-guide/) का पालन करें।
+
+2. फ्रंट मैटर में `शीर्षक` परिभाषित करें।। उदाहरण के लिए:
+    ```
+    ---
+    शीर्षक: सामग्री की तालिका में प्रदर्शित शीर्षक
```

Comment:
may be विषय सूची ?

Reply:
Where? I cannot find the reference of the word inside the file.

Reply:
Done